src/index.tsx

```diff
@@ -359,16 +360,19 @@ export class LikeCoinWalletConnector {
         if (!accessToken) {
           initiator = new Promise(resolve => {
             this._renderingRoot.render(
-              <AuthcoreDialog
-                onMount={({ containerId }) => {
-                  initAuthcore(this.options, { containerId });
-                }}
-                onClose={() => {
-                  this.closeDialog();
-                  resolve(undefined);
-                  this._events.emit('authcore_auth_closed');
-                }}
-              />
+              <IntlProvider language={language}>
+                <AuthcoreDialog
+                  onMount={({ containerId }) => {
+                    initAuthcore(this.options, { containerId });
+                  }}
+                  onClose={() => {
+                    this.closeDialog();
+                    resolve(undefined);
+                    this._events.emit('authcore_auth_closed');
+                  }}
+                  isHideSocialLogin={!!(this.options.authcoreClientId === 'likecoin-app-hidesocial')}
```

Comment:
Please name this `likecoin-app-hidesocial`  and `likecoin-app` as some constant.

Reply:
[🎨 Set the login social options as a constant](https://github.com/likecoin/likecoin-wallet-connector/pull/60/commits/8527d816b4ed2f3dd081028de36218dc10443689)